src/RootWithIntl.js

```diff
@@ -117,14 +116,9 @@ class RootWithIntl extends React.Component {
                       </AppCtxMenuProvider>
                     </MainContainer> :
                     <Switch>
-                      <TitledRoute
-                        name="changePassword"
-                        path="/reset-password/:token"
-                        component={<CreateResetPasswordLoader stripes={stripes} />}
-                      />
                       <TitledRoute
                         name="CreateResetPassword"
-                        path="/change-password/:token/:resetPasswordActionId"
+                        path="/change-password/:token"
```

Comment:
Back-end generates link with `/reset-password` path. Could this be changed?

Reply:
Done.